core/sequencer/server.go

```diff
@@ -380,3 +410,46 @@ func (s *Server) applyMutations(domainID string, mutatorFunc mutator.Func,
 	glog.V(2).Infof("applyMutations applied %v mutations to %v leaves", len(msgs), len(leaves))
 	return ret, nil
 }
+
+// HighWatermarks returns the total count and the highest watermark for each log.
```

Comment:
nit: "total count" is misleading - my first thought was total count of entries in each log. Maybe should be something like "total count of entries between the passed in low and the returned high watermarks"? Maybe swapping order of the returned values could help to make the comment shorter.

Reply:
hows "total count across all logs"